Server/JokesOnYou.Web.Api/DTOs/AllUsersDTO.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace JokesOnYou.Web.Api.DTOs
+{
+    public class AllUsersDTO
```

Comment:
I would recommend just having this as UserDTO or UserDetailDTO or MemberDTO
(if you want to use the userDTO you will have to remove the existing class and when we send back a token send an anonymous type)